src/__experimental__/components/form-field/form-field.component.js

```diff
@@ -48,6 +50,11 @@ const FormField = ({
   }
 
   const context = useContext(TabContext);
+  const largeScreen = useIsAboveBreakpoint(adaptiveLabelBreakpoint);
+  let inlineLabel = labelInline;
+  if (adaptiveLabelBreakpoint && labelInline) {
```

Comment:
```suggestion
  if (adaptiveLabelBreakpoint) {
```

I don't think this needs to be conditional based on `labelInline` I think adding the breakpoint is sufficient to enable the behaviour.